src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -202,6 +208,15 @@ async def setup_pex_cli_process(
     )
 
 
+def maybe_log_pex_stderr(stderr: bytes, pex_verbosity: int) -> None:
+    """Forward Pex's stderr to a Pants logger if conditions are met."""
+    log_output = stderr.decode()
+    if log_output and pex_verbosity > 0:
+        logger.info("%s", log_output)
+    elif log_output and "PEXWarning:" in log_output:
+        logger.warning("%s", log_output)
```

Comment:
I suggest reversing the order of the checks here, so we consistently log PEXWarnings as warnings, and not as info if pex verbosity is tuned up as that's just weird.